### DISCUSSION TITLE
Unit of area in NSGA 2 and SWMM input from Pyswmm data

### DISCUSSION THREAD

**Discussion by MohammadTahirGhori** [pyswmm/pyswmm#318] [Q&A]
I am working on an optimization process of NBS using SWMM LID measures, Pyswmm package and NSGA 2 Algorithm. I need to get pareto plots of optimization of reduction of flood, minimization of cost and environmental pollution. 
The NSGA 2 algorithm use the area of application of NBS measures as decision variables. 
I have a previous code the was written for the same purpose but different objectives. In the code I do not know what is the unit of area that NSGA 2 generate and what is the unit that SWMM need as input file for area. 
Can you help me understand the unit of area in SWMM and Algorithm. 
Thanks in advance.

**Comment by jennwuu** [comment]
Hello @MohammadTahirGhori,

Are you asking about [unit_area](https://pyswmm.readthedocs.io/en/v1.1.1/reference/lidgroups.html?highlight=unit_area#pyswmm.lidgroups.LidUnit.unit_area) from `LidUnit` ? 

The unit_area represents the area of a single LID representation in the SWMM model. The unit for LID unit area can be `ft2` or `m2`; the unit depends on the system unit. 

[SWMM Reference Manual Volume III—Water Quality (Includes the LID Module references)](https://nepis.epa.gov/Exe/ZyPURL.cgi?Dockey=P100P2NY.txt) is a good resource to understand hydrology in SWMM (including the process used for LID).
